src/Ui/Component/Listing/Column/OverviewPageActions.php

```diff
@@ -70,6 +75,20 @@ public function prepareDataSource(array $dataSource)
                         ]
                     ];
                 }
+
+                if (!empty($item['store_ids'])) {
+                    $store_ids = explode(',', $item['store_ids']);
```

Comment:
Please add a check on the store_ids property

Reply:
Issue is resolved. Added check on line below this statement